### PR TITLE
First release

### PR DESCRIPTION
**Pull request by evansiroky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# geocoder-argis-geojson
```
Comment (ansoncfit) [inline README.md]: geocoder-arcgis-geojson

File: README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+Isomorphic geocoding for reuse across our JavaScript libraries. This library is built to match the API of [Isomorphic Mapzen Search](https://github.com/conveyal/isomorphic-mapzen-search/) in order to facilitate a quick conversion of our applications to use ESRI's geocoder.  Read the ESRI docs [here](https://developers.arcgis.com/labs/rest/get-an-access-token/). Coordinates must be anything that can be parsed by [lonlng](https://github.com/conveyal/lonlng).
+
```
Comment (ansoncfit) [inline README.md]: Is it a joke that this redirects to https://github.com/conveyal/lonlat?

Comment (evansiroky) [reply README.md]: We used to call it that, but then it got changed to what it is now.  @trevorgerhardt can give some more history if need be.  :)

Comment (ansoncfit) [reply README.md]: I love that a repo with a snarky readme about lat/lon normalization has a redirect!

File: index.js [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    boundary.rect.maxLat,
+    boundary.rect.maxLon,
+    boundary.rect.minLat
+  ].join(',')
+}
+
+/**
+ * To make flow happy
+ */
+function coerceNullToString (s: ?string) {
```
Comment (trevorgerhardt) [inline index.js]: I've run into this too...are we doing something wrong? I feel like this is a common pattern.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+  return s || ''
+}
+
+function getGeocoder (clientId: ?string, clientSecret: ?string, endpoint: ?string) {
```
Comment (trevorgerhardt) [inline index.js]: Have you run into a use case where you have more than one ArcGIS geocoder?

Comment (evansiroky) [reply index.js]: Not really, but with the way I setup react-select-geocoder-arcgis the keys are sent as props, so theoretically they could change.

File: index.js (hunk 3 above)
Comment (evansiroky) [reply index.js]: Looks like there has been a debate on this issue: https://github.com/facebook/flow/issues/2814  Maybe I should use toString() instead?

**Pull request merged**